cmd/geth/main.go

```diff
@@ -326,6 +328,11 @@ func geth(ctx *cli.Context) error {
 
 	prepare(ctx)
 	stack, backend := makeFullNode(ctx)
+
+	if backend.ChainConfig().IsL2Migration(backend.CurrentBlock().Number()) {
```

Comment:
not sure if this would actually work...

when you start a node are you're not synced or out of sync... you don't know the block number

Reply:
removed this